sematic/ui/src/types/Types.tsx

```diff
@@ -825,6 +828,41 @@ function S3Button(props: {region?: string, bucket: string, location?: string}) {
   </>;
 }
 
+function ImageValueView(props: ValueViewProps) {
+  const { valueSummary } = props;
+  const { bytes, mime_type } = valueSummary;
+
+  const { user } = useContext(UserContext);
+
+  const [ imageBase64, setImageBase64 ] = useState<string | undefined>(undefined);
+
+  useEffect(() => {
+    const headers: HeadersInit = new Headers();
+    if (user?.api_key) {
+      headers.set("X-API-KEY", user.api_key);
+    }
+    fetch(`/api/v1/storage/blobs/${bytes.blob}/data`, {headers: headers})
```

Comment:
1. let's use the `useHttp()` hook, so we will have a chance to tweak http settings in a centralized way.  (Eg. add automatic caching)

2. Let's use `async` `await` for cleaner, synchronized style code.  (eliminating `then()`s)